quantum/rgb_matrix/animations/starlight_smooth_anim.h

```diff
@@ -0,0 +1,33 @@
+#ifdef ENABLE_RGB_MATRIX_STARLIGHT_SMOOTH
+RGB_MATRIX_EFFECT(STAR_LIGHT_SMOOTH)
+#    ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
+
+bool STAR_LIGHT_SMOOTH(effect_params_t* params) {
+    RGB_MATRIX_USE_LIMITS(led_min, led_max);
+
+    static uint16_t time_offsets[RGB_MATRIX_LED_COUNT] = {0};
+    static bool     initialized                        = false;
+
+    if (!initialized) {
+        for (uint8_t i = 0; i < RGB_MATRIX_LED_COUNT; i++) {
+            time_offsets[i] = rand() % 65500;
```

Comment:
Consider using the smaller (and possibly faster) lib8tion prng:
```suggestion
            time_offsets[i] = random16(65500);
```